include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -161,24 +161,24 @@ struct data_converter<CArray,
 };
 
 // Generic container converter
-template <typename Container, typename T = typename inspector<Container>::base_type>
+template <typename Container, typename T = typename inspector<Container>::element_type>
 struct container_converter {
-    typedef T value_type;
+    using value_type = T;
```

Comment:
Seems strange to alias for a longer name...

Reply:
alias with a name having a meaning ;)

I don't fix `data_converter` for now, i'm trying to remove what is not generic from them.

I cannot do everything in the same time, because it's a lot of little things locally *fixed*.

Reply:
Sure ;) I just didn't want to not say.